model/Core/Vocabularies/ProfileIdentifierType.md

```diff
@@ -4,28 +4,38 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-Enumeration of the valid profiles. 
+Enumeration of the valid profiles.
 
 ## Description
 
 There are a set of profiles that have been defined by a profile team.
-A profile consists of a namespace that may add properties and classes to the core profile unique to the domain covered by the profile.
-The profile may also contain additional restrictions on existing properties and classes defined in other profiles.
-If the creator of an SPDX collection of elements includes a profile in the list of conformanceProfiles, they are claiming that all contained elements conform to all restrictions defined for that profile.
+
+A profile consists of a namespace that may add properties and classes to the
+core profile unique to the domain covered by the profile.
+
+The profile may also contain additional restrictions on existing properties and
+classes defined in other profiles.
+
+If the creator of an SPDX collection of elements includes a profile in the list
+of conformanceProfiles, they are claiming that all contained elements conform
```

Comment:
The property is called `profileConformance`

Reply:
Thank you